requirements.txt

```diff
@@ -33,3 +33,4 @@ django-cors-headers==1.0.0
 
 # Zep 12 dependency
 django-uuslug==1.0.3
+watchdog==0.8.3 # use last non-bc-breaking version
```

Comment:
Bah en fait le débat a été long car nous n'avons pas tous eu les
disponibilités adéquates et qu'au départ il y avait des bugs (50 messages
dans la discussion, la moitiés pointaient des problèmes dans moncode).

On aurait pu être plus "rapide" d'au max 5/6 messages. La frustration vient
surtout du fait qu'on sent bien que générer les pdf and co de manière
asynchrone est primordial pour les perfs et l'utilisabilité du site.

Le 14 janvier 2016 à 11:27, firm1 <notifications@github.com> a écrit :

> In requirements.txt
> <https://github.com/zestedesavoir/zds-site/pull/3260#discussion_r49709501>
> :
>
> > @@ -33,3 +33,4 @@ django-cors-headers==1.0.0
> >
> >  # Zep 12 dependency
> >  django-uuslug==1.0.3
> > +watchdog==0.8.3 # use last non-bc-breaking version
>
> Tant que j'y pense, les débats seraient sans doute plus courts et sereins
> si chacun apportait des arguments dès le départ (cas d'utilisation,
> exemples, sources, …) lors de la détection des problèmes potentiels ou de
> la critique des remarques apportées.
>
> Ah ça c'est un peu de ma faute, M'en excuse. Pour bien faire il aurait
> fallu que je prenne le temps le lire le change de la PR pour éviter une
> série de use case complètement décalé, et ça j'ai eu la flemme :(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zestedesavoir/zds-site/pull/3260/files#r49709501>.
>